capstone-frontend/src/components/CommentCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+import '../styles/Groups.css';
+
+
+class CommentCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {}
+    }
+  }
+
+  fetchUserData = async () => {
+    await fetch(`/api/user?id=${this.props.comment.userID}`)
+        .then(response => response.json()).then(response => this.setState({user: response}))
+        .catch(function(err) {
+            //TODO #61: Centralize error output
+            alert(err); 
+        });
+  }
+
+  componentDidMount() {
+    this.fetchUserData();
+  }
+
+  render() {
+    const user = this.state.user && <div> {this.state.user.fullName}: </div>;
+    return (
+      <div className="comment-border">
+        {user}
+        <div> {this.props.comment.text} </div>
+        <div> {this.props.comment.whenCreated} </div>
```

Comment:
We should discuss the plan for dealing with Dates and determining whether we want to convert them all to a standard time (like UTC) in the database, then convert into the user's local time when displayed on the page.

Reply:
That seems like a good solution. I know the function `toUTCString()` allows us to convert a date in UTC time. We can use this before storing in the backend, and then on retrieving we can cast the string to a `Date` and then display it in the users current timezone.

Reply:
+1, it's great that you're thinking about scaling to different timezones!!